Documentation/Topics/CustomContentElements.rst

```diff
@@ -18,6 +18,9 @@ Search
 
 __________________________________________________
 
+Blogpostings (en)
+   * 2020-09-18, T3Terminal: `Learn TYPO3 Custom Elements with Core Ways
```

Comment:
The title and URL do not match.

Reply:
@sypets Oh Sorry, My mistake, Here is the correct URL https://t3terminal.com/blog/typo3-custom-elements/